<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="SCRMSHIN-VDI-11" name="utility.Utility" tests="1" failures="0" timestamp="16 May 2017 12:30:20 GMT" time="12.904" errors="1">
  <testcase name="endReport" time="12.904" classname="utility.Utility">
    <error type="java.lang.RuntimeException" message="javax.mail.MessagingException: Unknown SMTP host: smtp-relay.infor.com;
  nested exception is:
	java.net.UnknownHostException: smtp-relay.infor.com">
      <![CDATA[java.lang.RuntimeException: javax.mail.MessagingException: Unknown SMTP host: smtp-relay.infor.com;
  nested exception is:
	java.net.UnknownHostException: smtp-relay.infor.com
	at utility.Utility.execute(Utility.java:529)
	at utility.Utility.endReport(Utility.java:114)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:350)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: javax.mail.MessagingException: Unknown SMTP host: smtp-relay.infor.com;
  nested exception is:
	java.net.UnknownHostException: smtp-relay.infor.com
	at com.sun.mail.smtp.SMTPTransport.openServer(SMTPTransport.java:1280)
	at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:370)
	at javax.mail.Service.connect(Service.java:275)
	at javax.mail.Service.connect(Service.java:156)
	at javax.mail.Service.connect(Service.java:105)
	at javax.mail.Transport.send0(Transport.java:168)
	at javax.mail.Transport.send(Transport.java:98)
	at utility.Utility.execute(Utility.java:523)
	... 20 more
Caused by: java.net.UnknownHostException: smtp-relay.infor.com
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at com.sun.mail.util.SocketFetcher.createSocket(SocketFetcher.java:232)
	at com.sun.mail.util.SocketFetcher.getSocket(SocketFetcher.java:189)
	at com.sun.mail.smtp.SMTPTransport.openServer(SMTPTransport.java:1250)
	... 27 more
]]>
    </error>
  </testcase> <!-- endReport -->
</testsuite> <!-- utility.Utility -->
